inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/ParseWikidata.java

```diff
@@ -867,6 +887,7 @@ public static void main(String[] args) throws Exception {
                 .configure(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES, true)
                 .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         var lexParser = new ParseWikidata(parserOptions);
+        LexemesJsonDeserializer.setLanguage(parserOptions.locales);
```

Comment:
I couldn't figure out how to register a module in Jackson with parameters. So I did it this way with a global variable.